modulefiles/RDAS/orion.intel.lua

```diff
@@ -83,7 +83,7 @@ setenv("RDASAPP_TESTDATA","/work2/noaa/da/cmartin/CI/RDASApp/data")
 setenv("RDAS_RRFS_DATA_ROOT","/work/noaa/fv3-cam/tlei/dr-emc-rdas/")
 prepend_path("PATH","/apps/contrib/NCEP/libs/hpc-stack/intel-2018.4/prod_util/1.2.2/bin")
 
-execute{cmd="ulimit -s unlimited",modeA={"load"}}
+--cltorg execute{cmd="ulimit -s unlimited",modeA={"load"}}
```

Comment:
In my  rdasapp building, this line would cause building fail with message saying such changes are not allowed. 
I am wondering if it is a general issue or it was caused by my local env including the login nodes.

Reply:
do you already set the stack size in your login environment?

Reply:
@CoryMartin-NOAA , that is the tricky one.  I typed the above explanation of mine before seeing your question. 
If GDASapp has this command and no issue.  it shouldn't occur to rdasapp.

Reply:
Yeah it's been there for a while with no complaints: https://github.com/NOAA-EMC/GDASApp/blob/3bb46b61e710a77e85916d9b20388df1e1f390f3/modulefiles/GDAS/orion.intel.lua#L86 

Probably good for other reviewers to chime in before deciding to keep/remove

Reply:
It is preferable to be consistent with GDASApp.